spec/solver_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'spec_helper'
+
+require 'yaml'
+
+describe Solver do
+  before :each do
+    @solver = Solver.new
+  end
+  it 'Create new instance of solver class' do
+    @solver.should be_an_instance_of Solver
+  end
+
+  it 'Return the factorial of the number without breaking' do
```

Comment:
- [ ] Your tests for the factorial method should also include a test for Negative value which raises an exception as an expectation